src/settings/useSettingsApp.jsx

```diff
@@ -31,5 +31,10 @@ export default function useSettingsApp() {
 			label: PRODUCT_NAME,
 			component: SettingsView
 		});
+		upsertApp({
+			name: CHATS_APP_ID,
+			display: CHATS_APP_ID,
+			description: 'Chats Module for Zextras Carbonio'
```

Comment:
description and display should be translated.
display should not be the CHATS_APP_ID